Packs/Use_Case_Builder/Scripts/CreateUseCaseStepsList/CreateUseCaseStepsList.yml

```diff
@@ -0,0 +1,25 @@
+commonfields:
+  id: CreateUseCaseStepsList
+  version: -1
+contentitemexportablefields:
+  contentitemfields:
+    definitionid: ""
+    fromServerVersion: ""
+    itemVersion: ""
+    packID: ""
+    packName: generator
+    prevname: ""
+    toServerVersion: ""
+dockerimage: demisto/python3:3.10.11.57293
+enabled: true
+engineinfo: {}
+mainengineinfo: {}
+name: CreateUseCaseStepsList
+pswd: ""
+runas: DBotWeakRole
+runonce: false
+script: ''
+scripttarget: 0
+subtype: python3
+tags: []
+type: python
```

Comment:
Please add fromversion to all yml files in the pack describing the minimal XSOAR version on which the item will be used.

Reply:
Not sure i follow? Like what XSOAR version

Reply:
Yes.
Can find further explanation in [here](https://xsoar.pan.dev/docs/integrations/yaml-file#version-and-tests).